specs/src/specs/consensus.md

```diff
@@ -100,7 +99,7 @@ TODO
 
 The Tendermint consensus protocol is fork-free by construction under an honest majority of stake assumption.
 
-If a block has a [valid commit](#blocklastcommit), it is part of the canonical chain. If equivocation [evidence](./data_structures.md#evidence) is detected for more than 1/3 of voting power, the node must halt. See [rationale doc](../rationale/fork_choice_das.md) for more information.
+If a block has a [valid commit](#blocklastcommit), it is part of the canonical chain. If equivocation evidence is detected for more than 1/3 of voting power, the node must halt. See [rationale doc](../rationale/fork_choice_das.md) for more information.
```

Comment:
Equivocation detection happens per vanilla Tendermint so perhaps this is helpful: https://medium.com/tendermint/different-types-of-evidence-in-tendermint-5de4440fdd54

Reply:
Thank you for sharing the link. My main question concerns the following statement: 
> "equivocation evidence is detected for more than 1/3 of voting power." 

The detection mechanism is not explained for cases where no evidence data is available. The old specifications indicate that equivocation detection is aided by the presence of "evidence" data. However, since the evidence data has been removed, I believe that the detection mechanism should be revised accordingly. Unless there was no such explanation before, in which case no changes are necessary.

Reply:
perhaps we need to put this in the light client specs? iirc, they need to download the evidence manually anyway, which was the reason for removing it from the square

Reply:
or maybe we want to add that reasoning here? I'm unsure, but will defer to the author

Reply:
Agree that if there is to be further spec update to elaborate on the equivocation detection without evidence data, it can happen in a separate PR.